app/MemberProject.php

```diff
@@ -8,10 +8,14 @@
 
 class MemberProject extends Model
 {
+    protected $table="member_projects";
```

Comment:
This is a code style error, there should be spaces around `=`

Reply:
yep sr, i have fix it